tests/benchmarks/variable_length_filter.yml

```diff
@@ -0,0 +1,23 @@
+name: "VARIABLE-LENGTH-FILTER"
+remote:
+  - setup: redisgraph-r5
+  - type: oss-standalone
+dbconfig:
+  - dataset: "datasets/highly_connected.rdb"
+clientconfig:
+  - tool: redisgraph-benchmark-go
+  - parameters:
+    - graph: "g"
+    - rps: 0
+    - clients: 32
+    - threads: 4
+    - connections: 32
+    - requests: 1000000
+    - queries:
+      - { q: "MATCH (a)-[e*]->(b) WHERE e.connects = \"CD\" RETURN a.name, b.name", ratio: 0.25 }
+      - { q: "MATCH (a)-[e*]->(b) WHERE e.connects = \"CD\" OR e.connects = \"DE\" RETURN a.name, b.name", ratio: 0.25 }
+      - { q: "MATCH (a)-[e*]->(b) WHERE e.connects = \"CD\" OR e.connects = \"DE\" OR e.connects = \"EF\" RETURN a.name, b.name", ratio: 0.25 }
+      - { q: "MATCH (a)-[e*]->(b) WHERE e.connects = \"CD\" OR e.connects = \"DE\" OR e.connects = \"EF\" OR e.connects = \"FG\" RETURN a.name, b.name", ratio: 0.25 }
+kpis:
+  - le: { $.OverallClientLatencies.Total.q50: 1.0 }
+  - ge: { $.OverallQueryRates.Total: 25000 }
```

Comment:
remind me what's the meaning of `OverallQueryRates.Total` ?

Reply:
overall command rate in RPS